app/helpers/manageiq/providers/cisco_intersight/toolbar_overrides/physical_servers_center.rb

```diff
@@ -0,0 +1,57 @@
+module ManageIQ
+  module Providers
+    module CiscoIntersight
+      module ToolbarOverrides
+        class PhysicalServersCenter < ::ApplicationHelper::Toolbar::Override
+          button_group(
+            'physical_server_policy_choice',
+            [
+              select(
+                :physical_server_lifecycle_choice,
+                nil,
+                t = N_('Cisco Intersight'),
+                t,
+                :enabled => true,
+                :items   => [
+                  api_button(
+                    :physical_server_decommission,
+                    nil,
+                    N_('Decommission server'),
+                    N_('Decommission'),
+                    :icon    => "pficon pficon-off fa-lg",
+                    :api     => {
+                      :action => 'decommission_server',
+                      :entity => 'physical_server'
+                    },
+                    :confirm => N_("Decommission selected servers?"),
+                    :enabled => true,
+		    :onwhen  => "1+",
```

Comment:
I'm qualified to review this part :laughing: looks like maybe a mixture of tabs/spaces (same down on +43)